packages/keystone/src/___internal-do-not-use-will-break-in-patch/admin-ui/pages/ItemPage/index.tsx

```diff
@@ -253,15 +253,17 @@ function DeleteButton({
           confirm: {
             label: 'Delete',
             action: async () => {
-              await deleteItem().catch(err => {
-                toasts.addToast({
+              try {
+                await deleteItem();
+              } catch (err) {
+                return toasts.addToast({
```

Comment:
This error was also occurring on the item view. 
Early return here to stop the route changing and success toast from being rendered.